mmedit/datasets/pipelines/augmentation.py

```diff
@@ -145,6 +148,190 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class RandomResizedCrop(object):
+    """Crop the given numpy array to random size and aspect ratio.
+
+    A crop of random size (default: of 0.08 to 1.0) of the original size and a
+    random aspect ratio (default: of 3/4 to 4/3) of the original aspect ratio
+    is made. This crop is finally resized to given size.
```

Comment:
A crop of a random proportion of the original image and a random aspect ratio of the original aspect ratio is made. The cropped image is finally resized to a given size specified by 'crop_size'.